### DISCUSSION TITLE
Change date and currency format

### DISCUSSION THREAD

**Discussion by ivancarlosti** [firefly-iii/firefly-iii#9201] [Firefly III] [answered]
Please, there is any way to change date and currency format?

Even setting up .env to `DEFAULT_LOCALE=pt_BR`, the crazy USA format persist on forms field (it displays correctly, but I need to type the values in USA format)

I am running Firefly III by docker compose and when I [run](https://docs.firefly-iii.org/how-to/firefly-iii/advanced/locales/) `locale -a` on my system it returns `pt_BR`

**Comment by JC5** [comment, marked as answer]
Hi, good question!

That's a browser setting, not a Firefly III setting. Date and number fields follow the locale of your browser. I can't change that.
